frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/import/import_panel.dart

```diff
@@ -162,6 +132,55 @@ class ImportPanel extends StatelessWidget {
 
     importCallback(importType, '', null);
   }
+
+  Widget _buildKeyboardListeningFlowyContainer(
+    BuildContext context,
+    double height,
+    double width,
+  ) {
+    return KeyboardListener(
+      autofocus: true,
+      focusNode: FocusNode(),
+      child: FlowyContainer(
+        Theme.of(context).colorScheme.surface,
+        height: height,
+        width: width,
+        child: GridView.count(
+          childAspectRatio: 1 / .2,
+          crossAxisCount: 2,
+          children: ImportType.values
+              .where((element) => element.enableOnRelease)
+              .map(
+                (e) => Card(
+                  child: FlowyButton(
+                    leftIcon: e.icon(context),
+                    leftIconSize: const Size.square(20),
+                    text: FlowyText.medium(
+                      e.toString(),
+                      fontSize: 15,
+                      overflow: TextOverflow.ellipsis,
+                      color: Theme.of(context).colorScheme.tertiary,
+                    ),
+                    onTap: () async {
+                      await _importFile(parentViewId, e);
+                      if (context.mounted) {
+                        FlowyOverlay.pop(context);
+                      }
+                    },
+                  ),
+                ),
+              )
+              .toList(),
+        ),
+      ),
+      onKeyEvent: (event) {
+        if (event is KeyDownEvent &&
+            event.physicalKey == PhysicalKeyboardKey.escape) {
+          FlowyOverlay.pop(context);
+        }
+      },
```

Comment:
`child` properties should be listed last, put this at the top just below the `focusNode,` property